### PR TITLE
feat: changed layout of individual user cards in user management page

### LINKED ISSUE #2435: Revamp User Management UI
qwopdwk

the UI is having lots of unwanted use of blank space, can a better UI for the User management page!

### PR DESCRIPTION
**Pull request by Pragati1610**
Fixes #2435 

- Removed the extra space in each card
- Changed the look of individual facilities
- Made 3 columns instead of 2 columns

Before:
image

After
image

### CODE REVIEW

**Comment by nihal467** [comment]
test approved

**Comment by sainak** [comment]
@Pragati1610 delete the package-lock from your branch as we have started using yarn.

File: src/Components/Common/ClosingTag.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+function ClosingTag() {
+  return (
+    <span className="cursor-pointer font-glight">
+      <svg
+        v-else-if="name === 'close'"
```
Comment (developedBySJ) [inline src/Components/Common/ClosingTag.tsx]: Remove this line as we are using `React` not `vue`

File: src/Components/Common/ClosingTag.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+function ClosingTag() {
```
Comment (developedBySJ) [inline src/Components/Common/ClosingTag.tsx]: For Icon, You can use
- Material icons
- Font awesome icons

**Review by developedBySJ — CHANGES_REQUESTED** [review]
**The Layout is not responsive!**

Use 3 col layout for `lg` breakpoint

image

File: src/Components/Common/ClosingTag.tsx (hunk 2 above)
Comment (Pragati1610) [reply src/Components/Common/ClosingTag.tsx]: made the changes

**Review by developedBySJ — APPROVED** [review]
LGTM

**Pull request merged**